### ISSUE TITLE
Request: Table to show the top players that kill new people.

### ISSUE THREAD

**Issue by TomLewis** [plan-player-analytics/Plan#1262]
We have protection on new players that lasts a few hours, but we have been having an issue with some long term players making note of who's new when they join, and waiting those few hours to hunt them down and kill them.
This is terrible for keeping those new players you have put a lot of effort into getting onto the server, so much effort is needed to get people to see your server, let alone get them to join and play, and then only to be ruined by some asshole long term player.

Plan records all this information, time joined how long their sessions are etc, and PvP. The data already exists, it just needs to be displayed.
if we could have a PvP table somewhere that shows the top offenders of killing people that have joined for the first time, we could moderate this behavior with these top offenders. 

Even more there could be a filter to show how many of these people didn't come back to the server after being killed, so for example Player1 Kills player501 after player501 has only been playing for 3 hours and hes not come back to the server since, then this is classed as a lost player.

**Comment by Rsl1122** [comment]
I got an idea for an alternative (possibly cheaper to calculate), where the kill list tables would show player name in light green or a register symbol (similar to sessions list) if the death occurred within 24h of the player's register date. This would be visible in both server and player page *PvP & PvE* tab, so looking at a player's page should make it clear who is killing new players.

----

It's unfortunately very difficult to deduce cause for players not returning, At some point I tried to find data points that could be used to try guess why a player stopped playing, but the amount of wrong guesses would make the guesses worthless (and might point people to wrong direction).

A classification for a death could be calculated, "Rage quit" (for lack of a better term), where player's session ends soon after they were killed by another player. I was going to add this during 5.0, but it was cut to release sooner. I'll probably need a better term for it.
